packages/Uploader/src/components/DropZone/DropZone.styles.js

```diff
@@ -2,6 +2,30 @@ import stylers from "@paprika/stylers";
 import tokens from "@paprika/tokens";
 import styled, { css } from "styled-components";
 
+export const Container = styled.div`
+  label {
+    cursor: pointer;
+    display: inline-block;
+    height: 100%;
+    position: absolute;
+    width: 100%;
+  }
+
+  input[type="file"] {
```

Comment:
I don't think you still need this block since it's already display:none https://github.com/acl-services/paprika/pull/1068/files#diff-a1b648477dcc45faf5b5cbf86adb24f032348ccf409c77a839a32cb21e397736R71

Reply:
When we discussed this on Friday, I didn't realize what the original strategy was, but I think we should keep it.  Basically the `visually hidden` `<input type=file>` is what will receive keyboard focus, but when it does, the `<Button>` should become highlighted as if IT has keyboard focus.  It's a trick, but interacting with the `<input>` is going to work better for AT (assistive technology – namely screen readers).  But then we want the screen reader to ignore the actual `<Button>`.  This is why there was some weird `tabIndex` values... and in fact we should retain the `tabIndex={-1}` on the `<Button>`.

Reply:
Discussed with Mike offline, should be something like this https://codesandbox.io/s/ecstatic-sun-b121k?file=/src/App.js